modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm

```diff
@@ -0,0 +1,122 @@
+#define EMERGENCY_SUIT_MIN_TEMP_PROTECT 237
+#define EMERGENCY_SUIT_MAX_TEMP_PROTECT 100
+
+
+// The suit
+/obj/item/clothing/suit/space/emergency
+	name = "emergency space suit"
+	desc = "A fragile looking emergency spacesuit for limited use in space."
+	icon_state = "syndicate-orange"
+	inhand_icon_state = "syndicate-orange"
+	heat_protection = NONE
+	min_cold_protection_temperature = EMERGENCY_SUIT_MIN_TEMP_PROTECT
+	max_heat_protection_temperature = EMERGENCY_SUIT_MAX_TEMP_PROTECT
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 20, FIRE = 0, ACID = 0)
+	clothing_flags = STOPSPRESSUREDAMAGE | SNUG_FIT
+	actions_types = null
+	show_hud = FALSE
+	max_integrity = 100
+	slowdown = 3
+	/// Have we been damaged?
+	var/torn = FALSE
+
+/obj/item/clothing/suit/space/emergency/equipped(mob/user, slot)
+	. = ..()
+	RegisterSignal(user, COMSIG_MOB_APPLY_DAMAGE, .proc/user_damaged)
+
+/obj/item/clothing/suit/space/emergency/dropped(mob/user)
+	. = ..()
+	UnregisterSignal(user, COMSIG_MOB_APPLY_DAMAGE)
+
+/obj/item/clothing/suit/space/emergency/proc/user_damaged(datum/source, damage, damagetype, def_zone)
+	SIGNAL_HANDLER
+	if(damage && !torn && prob(50))
+		balloon_alert_to_viewers("[src] tears!")
+		clothing_flags &= ~STOPSPRESSUREDAMAGE
+		torn = TRUE
+		playsound(src, 'sound/weapons/slashmiss.ogg', 50, TRUE)
+		playsound(src, 'sound/effects/refill.ogg', 50, TRUE)
+		update_appearance()
+
+
+/obj/item/clothing/suit/space/emergency/update_name(updates)
+	. = ..()
+	if(torn)
+		name = "torn [src]"
+
+/obj/item/clothing/suit/space/emergency/examine(mob/user)
+	. = ..()
+	if(torn)
+		. += span_danger("It looks like it has been torn and is completely useless!")
+
+#undef EMERGENCY_SUIT_MIN_TEMP_PROTECT
+#undef EMERGENCY_SUIT_MAX_TEMP_PROTECT
+
+#define EMERGENCY_HELMET_MIN_TEMP_PROTECT 2.0
```

Comment:
why not just have these defines at the top of the file and undef them all at the bottom

Reply:
i guess